build.bat

```diff
@@ -375,13 +401,46 @@ if !BUILD_CPP! == 1 (
 	-additionalHeaderFiles "!additional_header_files!"^
 	-additionalSourceFiles "!additional_source_files!"^
 	-additionalDefines "!ADDITIONAL_DEFINES!"^
+	!additional_rti_libs!^
 	!rtiddsgen_extra_options! !additional_defines_custom_type!^
 	-d "%classic_cpp_folder%" "%idl_location%\perftest.idl"
 	if not !ERRORLEVEL! == 0 (
 		echo [ERROR]: Failure generating code for %classic_cpp_lang_string%.
 		call::clean_src_cpp_common
 		exit /b 1
 	)
+
+	if !FLATDATA_AVAILABLE! == 1 (
+		@REM # TODO: REMOVE THIS
```

Comment:
Add a comment instead explaining why we do this, check with Aida if we have a bug ID in Jira for this, else, make sure we create a Jira issue and add here that Jira issue.